### DISCUSSION TITLE
Clean up auxiliary files does not remove .aux files

### DISCUSSION THREAD

**Discussion by lambdakilo** [James-Yu/LaTeX-Workshop#4163] [Q&A]
No errors or warnings on Output. Any ideas what's up? Before this worked for me but now the command does not remove the .aux files.
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/31825085/c48f6bd0-01a7-4881-89d5-9dcbd0ee67a3)

**Comment by jlelong** [comment]
It is more likely related to `latexmk` that the extension. From your logs, cleaning is done by calling `latexmk -c file.tex`. You can try to run this command directly from a shell and see what is says.

If it is not a change in `latexmk`, then maybe you were using the `glob` method to clean files, see https://github.com/James-Yu/LaTeX-Workshop/wiki/Compile#latex-workshoplatexcleanmethod

**Reply by lambdakilo** [reply]
Hmm you're right good observation. `latexmk -c Ch.20_Methods.tex` does indeed work but it only deletes that files .aux files. Is there a way to remove all temporary files created by LaTeX in that directory?

**Reply by jlelong** [reply]
With latexmk `4.82a`, everything works as expected for me. All `.aux` generated by `latexmk` are cleaned by calling `latexmk -c`

**Reply by lambdakilo** [reply]
Same for me apparently. I wonder why this extension only removes the .aux file of the currently opened file only.

**Reply by jlelong** [reply]
I confirm that all the `.aux` related to the build are indeed removed when calling clean from the extension.
There might be something wrong on your side.
